src/lib/tooltip/tooltip.ts

```diff
@@ -309,6 +310,10 @@ export class TooltipComponent {
       // If this was set to true immediately, then a body click that triggers show() would
       // trigger interaction and close the tooltip right after it was displayed.
       this._closeOnInteraction = false;
+
+      // Call change detection manually so tooltip will also work 
+      // if any parent component has set the ChangeDetectionStrategy to OnPush
+      this._ref.detectChanges();
```

Comment:
I think rather than always triggering a `ChangeDetection` we should just mark for a new check.

We are doing this in other components as well. 
> https://angular.io/docs/js/latest/api/core/index/ChangeDetectorRef-class.html